setup.py

```diff
@@ -7,7 +7,7 @@
     'README.rst')
 
 dependencies = [
-    'beautifulsoup4>=4.2.1,<4.5.0',
+    'beautifulsoup4==4.8.2',
```

Comment:
having a fixed version might cause incompatibility issues with other packages that require a different version of beautifulsoup, a >= is preferred to allow for flexibility.